phpunit.xml.dist

```diff
@@ -6,7 +6,10 @@
 >
     <testsuites>
         <testsuite name="Unit tests">
```

Comment:
I suggest naming the suites "unit" and "integration" so it could be started more easily from the CLI:

`vendor/bin/phpunit --testsuite unit`
`vendor/bin/phpunit --testsuite integration`